x/inflation/keeper/hooks_test.go

```diff
@@ -146,10 +156,29 @@ func (suite *KeeperTestSuite) TestPeriodChangesAfterEpochEnd() {
 			"[Period 1] with skipped epochs - period changes once enough epochs have passed",
 			1,
 			2*currentEpochPeriod + 11,
+			epochstypes.DayEpochID,
 			10,
 			true,
 			true,
 		},
+		{
+			"[SkippedEpoch 0] set DayEpochID disabledInflation",
```

Comment:
What does `[SkippedEpoch 0]` mean here? 

We use `[Period 0]` and `[Period 1]` above to signalize which period we are at.